src/components/PassphraseLink.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { FC } from 'react'
+
+import { openInWebBrowser } from '../utils/misc'
+import ActionLink from './ActionLink'
+
+const passphraseWikiLink = 'https://wiki.alephium.org/wallet/Desktop-Wallet-Guide/#passphrase'
+
+const PassphraseLink: FC = ({ children }) => (
+  <ActionLink onClick={() => openInWebBrowser(passphraseWikiLink)}>{children}</ActionLink>
+)
+
+export default PassphraseLink
```

Comment:
Nit: This as a file feels like DRYing too much

Reply:
Where would you put it? I don't want to repeat the URL in 2 files. So if I put it in one file, why not put the rest of the code as well so that I don't repeat that too?

Reply:
We could have one file where we define all URL constants used throughout the app. 
This would help with updating the links if needed, and avoid duplicating constant declaration.

Reply:
Sounds good to me!

Reply:
done